code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -92,3 +92,10 @@
 	var/hal_screwydoll
 	/// When an braindead player has their equipment fiddled with, we log that info here for when they come back so they know who took their ID while they were DC'd for 30 seconds
 	var/list/afk_thefts
+
+	///SKYRAT EDIT BEGIN. Oversized variables
```

Comment:
Since you're just declaring new variables, it should be done in a modular file, not in a /tg/ one.

Reply:
Got it. My bad